demo/style.css

```diff
@@ -29,93 +29,100 @@ pre {
 }
 
 code {
+  display: block;
   font-family: Consolas, Monaco, monospace;
   font-size: 10pt;
+  overflow-x: auto;
+  padding: 0.5em;
   scrollbar-width: thin;
+  -moz-tab-size: 4;
+  tab-size: 4;
 }
 
-body pre code.hljs {
-  padding-left: 1.5em;
-  /* compatible with school-book */
-  padding-top:15px;
+aside {
```

Comment:
I'm not sure if we're fighting over styling here or if this is accidental.  I _very specifically_ tweaked this last time to add more padding... and almost didn't notice that you removed this... I've added it back.

Reply:
The top padding is compatible with school-book, but the left padding is not:
![image](https://user-images.githubusercontent.com/2564094/102653953-96d37280-4124-11eb-8d36-aa877ffb991c.png)

Having school-book style padding on other themes feels a bit odd (especially since it's only on the top and left)
![image](https://user-images.githubusercontent.com/2564094/102654018-b8ccf500-4124-11eb-97b1-ef6f5e1ab693.png)
![image](https://user-images.githubusercontent.com/2564094/102654033-bff40300-4124-11eb-9ad9-d3e28c918137.png)

I would suggest letting the theme set the padding.

Reply:
> Having school-book style padding on other themes feels a bit odd  

I don't find it problematic.  As a general rules themes should not enforce pixel-precise padding (and no other themes do).  I may even remove school book in v11 (or remove it's BG at least).  If a modern theme were submitted that did something similar I would reject it.

> I would suggest letting the theme set the padding.

No, sorry, the default is pretty awful IMHO.  I think I'll add it to the list of things to change in v11.